bigquery/dataset.go

```diff
@@ -298,6 +323,13 @@ func bqToDatasetMetadata(d *bq.Dataset, c *Client) (*DatasetMetadata, error) {
 	return dm, nil
 }
 
+func bqToStorageBillingModel(storageBillingModel string) StorageBillingModel {
```

Comment:
I worry that this kind of thing is brittle if the service introduces another billing model.